lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUDriver.cs

```diff
@@ -168,11 +169,26 @@ public void OnNext(IActiveContext activeContext)
                                 .Set(TaskConfiguration.Task,
                                     GenericType<UpdateTaskHost<TMapInput, TMapOutput, TResult>>.Class)
                                 .Build(),
-                            _configurationManager.UpdateFunctionConfiguration
+                            _configurationManager.UpdateFunctionConfiguration,
+                            _configurationManager.ResultHandlerConfiguration
                         })
                         .BindNamedParameter(typeof (InvokeGC), _invokeGC.ToString())
                         .Build();
 
+                try
+                {
+                    TangFactory.GetTang()
+                        .NewInjector(partialTaskConf, _configurationManager.UpdateFunctionCodecsConfiguration)
+                        .GetInstance<IIMRUResultHandler<TResult>>();
+                }
+                catch (Exception)
```

Comment:
This is not an exception. This is a way of checking whether user has provided configuration or not. Otherwise use identity. I will log though.